pact/coin-contract/v2/coin.repl

```diff
@@ -813,3 +813,22 @@
   "gas-guard succeeds when GAS not present, but keyset is"
   true
   (gas-guard (keyset-ref-guard "bez")))
+
+(commit-tx)
+
+(begin-tx)
+
+(use coin)
+
+(expect-failure
+  "Slashes fail without the presence of SLASH"
+  "not granted: (coin.SLASH)"
+  (slash "brandon" 1.0))
+
+(test-capability (SLASH))
+
+(env-keys ["brandon"])
+(expect
+  "Slashes succeed in the presence of SLASH"
+  "Write succeeded"
+  (slash "brandon" 1.0))
```

Comment:
Should cover negative amount, unit, valid account

Reply:
Addressed all of these